### DISCUSSION TITLE
Selecting Environment in Signoz

### DISCUSSION THREAD

**Discussion by Samuelseggs** [SigNoz/signoz#4869] [Q&A]
Hello,

I seem to have issues filtering by Environment in Service, 

I have tried to set deployment.environment=DEV in infra-otel-agent like this

![image](https://github.com/SigNoz/signoz/assets/7638851/545a59cb-529f-4dad-a70d-ce0ad72b66a1)

But this does not work

I'm using autoinstrumentation to implement OTEL into my code. here is what i got on the pod

![image](https://github.com/SigNoz/signoz/assets/7638851/4ae79b01-7494-41a3-8bf1-a4dd0324f1fa)

Here is my setup.

I'm pushing traces from multiple k8s cluster  to the cloud signoz environment. it's very difficult to trace which log belongs to their environment.

I can see the different logs because i added this --set global.clusterName=DEV to the helm chart. but i cannot see this in Service and Service Map

![image](https://github.com/SigNoz/signoz/assets/7638851/a9197235-5290-42e9-9359-124611d29612)

**Comment by Samuelseggs** [comment]
This issue was resolved

**Reply by func0der** [reply]
how though?
